### DISCUSSION TITLE
Can we have a new column with information about the citizenship requirement?

### DISCUSSION THREAD

**Discussion by zhu-yifang** [ReaVNaiL/New-Grad-2024#6] [Ideas]
I want to add a new column about the citizenship requirement because there are many jobs that require the applicant to be a US citizen or green card holder. And this is very important information for many people.

**Comment by ReaVNaiL** [comment]
Yeah I can work on adding an extra tab for each section and specify whether they will sponsors and any other relevant notes.

**Reply by zhu-yifang** [reply]
Thank you so much!

**Comment by echefulouis** [comment]
I noticed for Sentry.io, There is no visa sponsorship. maybe we could add that
